extensions/Worker.Extensions.Timer/src/Converters/TimerPocoConverter.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Timer.Converters
+{
+    internal sealed class TimerPocoConverter : IInputConverter
```

Comment:
If the only type we convert to is TimerInfo, is this better named as `TimerInfoConverter`? What other POCOS will the timer trigger convert?

Reply:
Renamed to `TimerInfoConverter`